src/main/java/frc/robot/commands/universalcommands/SetFlywheelRPM.java

```diff
@@ -0,0 +1,53 @@
+// Copyrights (c) 2018-2019 FIRST, 2020 Highlanders FRC. All Rights Reserved.
+
+package frc.robot.commands.universalcommands;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import frc.robot.*;
+
+public class SetFlywheelRPM extends CommandBase {
+
+    private static double velocity;
```

Comment:
Why are these static?

Reply:
That's static because it needed to be accessed in a static way in `isAtTargetRPM()`.

Reply:
This is just a really dangerous way to use static variables. Normally you should have an access guard that figures out if the variables have already been initialized or something. Essentially you can only have instance of `SetFlywheelRPM` in your entire codebase. This may be your desired use case but it is super hard to understand that by glancing at this class.

Is there any way for the object requiring `isAtTargetRPM` to have a pointer to the instance of `SetFlywheelRPM`?

Reply:
I can change `isAtTargetRPM()` to be a nonstatic method and then change `velocity` to nonstatic as well.